pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/MultiStageBrokerRequestHandler.java

```diff
@@ -179,6 +225,7 @@ private BrokerResponse handleRequest(long requestId, String query,
       stageIdStatsMap.put(stageId, new ExecutionStatsAggregator(traceEnabled));
     }
 
+    long executionStartTime = System.nanoTime();
```

Comment:
nit: rename it to `executionStartTimeNs` to make it clear. Same for the other newly added variables.